policyengine_us/variables/gov/states/mn/tax/income/credits/mn_child_and_working_families_credits.py

```diff
@@ -0,0 +1,62 @@
+from policyengine_us.model_api import *
+
+
+class mn_child_and_working_families_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Minnesota child and working families credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.revisor.mn.gov/statutes/cite/290.0661#stat.290.0661.4"
+        "https://www.revisor.mn.gov/statutes/cite/290.0671"
+        "https://www.revenue.state.mn.us/sites/default/files/2024-01/m1cwfc-23_1.pdf"
+    )
+    defined_for = StateCode.MN
+
+    def formula(tax_unit, period, parameters):
+        # Minnesota combines the Child Tax Credit and
+        # the Working Family Credit into a single credit
+        # The credits are defined by separate legal code sections
+        # but are calculated together in the tax forms.
+        # Child Tax Credit computation:
+        qualifying_children = tax_unit("ctc_qualifying_children", period)
```

Comment:
the legal code defines qualifying children without 18-year-olds. this is redundant but can we add it for completeness?

for posterity, it also removes the SSN requirement in [IRC 32(m)](https://www.law.cornell.edu/uscode/text/26/32#m), though we don't currently capture that